README.md

```diff
@@ -146,7 +152,7 @@ A notification signal will be triggered whenever a _new_ datafile is fetched and
 
```

Comment:
Nit, line 144: use block code style instead of in-line code style for `notification_center.add_notification_listener(Optimizely::NotificationCenter::NOTIFICATION_TYPES[:OPTIMIZELY_CONFIG_UPDATE], @callback)`